src/registrar/templates/domain_sidebar.html

```diff
@@ -34,7 +34,7 @@
             >
                 DNSSEC
             </a>
-            {% if domain.dnssecdata is not None or request.path|startswith:url and request.path|endswith:'data' %}
+            {% if domain.dnssecdata is not None or request.path|startswith:url and request.path|endswith:'dsdata' %}
```

Comment:
```suggestion
            {% if domain.dnssecdata is not None or (request.path|startswith:url and request.path|endswith:'dsdata') %}
```
For readability (if it lets you), you should wrap the connected or logic (or and?) in parenthesis. Not sure if the `or` is meant to be linked or if the `and` is... I assume the `and` statement though?